website/docs/components/form/text-input/partials/code/how-to-use.md

```diff
@@ -102,6 +102,17 @@ Use the `@isRequired` and `@isOptional` arguments to add a visual indication tha
 </Hds::Form::TextInput::Field>
 ```
 
+#### Character count
+
+To support users meet content length constrains set `@minLength` and/or `@maxLength` to a `CharacterCount` contextual component.
```

Comment:
This is a very basic example, that covers only one of the possible use cases. We should point to the base elements examples for details, or add the examples directly here (including the linking with the "validation" error too, it's not obvious or explained elsewhere).

[Unfortunately, as a consumer, it's hard to imagine how to use this component without some more extensive examples of the possible combinations/cases]

Reply:
Added more examples in efa65c4, 04ebfad, and aaa6c52